### PR TITLE
Do not use Autoprefixer for text-decoration property with CSS var as value

### PR DESCRIPTION
**Pull request by lex111**
I just noticed that Autoprefixer adds a vendor prefix for the text decoration property (`-webkit-text-decoration`), which uses the CSS var as its value, for example:

```
    -webkit-text-decoration: var(--ifm-link-hover-decoration);
    text-decoration: var(--ifm-link-hover-decoration)
```

 I'm not sure if this is an Autoprefixer bug, as this property has "extra" values that have little support yet. https://caniuse.com/?search=text-decoration

Since we are using "regular" values, we should explicitly disable Autoprefixer for this property.

### CODE REVIEW

File: packages/postcss-preset-infima/index.js [hunk 1]
```diff
@@ -22,7 +22,6 @@ const scss = require('postcss-scss');
 
 module.exports = (options) => ({
   plugins: [
-    autoprefixer(),
```
[the remark this answers is not in the document]
Comment (Simek) [reply packages/postcss-preset-infima/index.js:25]: So why you have left import for `autoprefixer` and package definition in the dependencies?

**Comment by lex111** [comment]
Apparently forgot to do this, thanks for letting me know, I'll fix it soon.

2021-02-17 18:58 GMT+03:00, Bartosz Kaszubowski <notifications@github.com>:

**Comment by Simek** [comment]
> Apparently forgot to do this, thanks for letting me know, I'll fix it soon. 2021-02-17 18:58 GMT+03:00, Bartosz Kaszubowski <notifications@github.com>:
> […](#)
> @Simek commented on this pull request. > @@ -22,7 +22,6 @@ const scss = require('postcss-scss'); module.exports = (options) => ({ plugins: [ - autoprefixer(), @lex111 So why you have left import for `autoprefixer` and package definition in the dependencies? -- You are receiving this because you were mentioned. Reply to this email directly or view it on GitHub: [#49 (comment)](https://github.com/facebookincubator/infima/pull/49#discussion_r577731904)

It's already fixed in #58

**Comment by Simek** [comment]
@lex111 I have also left few comments in various, already merged PR by you. It would be nice, if you can address the question and comments in there too.

**Pull request merged**